pwa/src/styles/main.css

```diff
@@ -230,4 +230,6 @@ a:hover {
     background: inherit !important;
 }
 
-
+span > svg{
+    margin-left: 5px;
+}
```

Comment:
Deze `css` gaat nu altijd en overal **elke svg in een span** selecteren, dat is niet de bedoeling. Twee dingen hoe we dit netjes kunnen oplossen:

1. Deze code mag naar een apart bestand (van het component waar we de Tooltip includen);
2. De `<Tooltip />` include mag een class meekrijgen (ik denk dat we daar nog niets voor hebben gebouwd, mag een `prop` worden genaamd `LayoutClassName`)